.github/workflows/crawl.yml

```diff
@@ -17,6 +17,7 @@ jobs:
 
       - name: Run the crawl script
         run: ./crawl.sh
+        continue-on-error: true
```

Comment:
I think the current crawl code accomplishes the same thing with [this line](https://github.com/cfpb/crawl-cfgov/blob/main/.github/workflows/crawl.yml#L24). It now continues the job even if the crawl task exits with an error code.